CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Contributing to EiffelVis
+Thank you for your interest in the project!
+
+This document explains the guidelines you should follow to smoothly contribute to EiffelVis.
+
+## Bug Reports
+Have you found a problem, or simply observed potentially incorrect behavior? [open an issue](https://github.com/ItJustWorksTM/EiffelVis/issues).
+
+If you have the time, look through already [existing issues](https://github.com/ItJustWorksTM/EiffelVis/issues?q=is%3Aissue). In case someone else has already reported your problem you can simply move on with your day or submit extra information about your problem.
+
+## Contributing
+Contributing to the project will most likely be in the form of a [pull request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests).
+So when you want to submit a bug fix or a documentation improvement, fork the repo, apply your changes and create a pull request. A maintainer of the project will look over the pull request and give feedback on how to it might be improved. When all feedback is resolved the PR will be accepted into the repository 
+
+In the case that you want to contribute larger amounts of work be sure to contact the project maintainers before hand to discuss on how to go about doing it.
```

Comment:
In case you want to contribute larger amounts of work, be sure ... maintainers beforehand to discuss the procedure.